src/components/FilterBar.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from "react";
+import { PropsType } from "../containers/FilterBarContainer";
+import * as objAssign from "object-assign";
+import { Store } from "azsearchstore";
+import { defaultCss } from "../utils/css";
+
+export type State = {};
+
+class FilterBar extends React.Component<PropsType, State> {
+  render() {
+    const { onClear, hasSelectedFacets } = this.props;
+    const text = "clear filter(s)";
+
+    let clearComponent = hasSelectedFacets ?
+      <a
+        href="#"
+        onClick={ e => {
+          e.preventDefault();
+          if (onClear) onClear();
+        }}
+      >{text}</a>
+    :
+      <span
+      className="text-muted"
```

Comment:
Adding this back in as it got hidden after the most recent iteration: 

Can you add an entry for these classNames in [src/utils/css.ts](https://github.com/EvanBoyle/AzSearch.js/blob/master/src/utils/css.ts). In order to allow others to customize/theme the controls as they see fit, I've tried to make CSS configurable wherever possible.  You can see how we [consume custom css here](https://github.com/EvanBoyle/AzSearch.js/blob/master/src/components/CheckboxFacet.tsx#L12), you'll also have to add the custom css as a prop, and as an optional input the the api.